cmd/cloud-controller-manager/main.go

```diff
@@ -157,5 +166,15 @@ func cloudInitializer(config *config.CompletedConfig) cloudprovider.Interface {
 		gceCloud.SetEnableRBSDefaultForL4NetLB(true)
 	}
 
+	if enableMultiNetworking {
+		gceCloud, ok := (cloud).(*gce.Cloud)
+		if !ok {
+			// Fail-fast: If enableMultiNetworking is set, the cloud
+			// provider MUST be GCE.
+			klog.Fatalf("multi-networking nodes require GCE cloud provider, but got %T", cloud)
+		}
+		gceCloud.SetEnableMultiNetworking(true)
```

Comment:
I know I shared https://github.com/kubernetes/cloud-provider-gcp/pull/808 with you as an example but the part which doesn't apply to this case here is that we don't need to propagate this information through the gceCloud object. 

Instead, I think we should define a flag at https://github.com/kubernetes/cloud-provider-gcp/blob/4c1826884778c9a6cbb2f43b7d17bbb00237516b/cmd/cloud-controller-manager/options/nodeipamcontroller.go#L42